README.md

```diff
@@ -22,13 +22,17 @@ At this step, the upgrader checks the project against a set of rules. The main g
 
 To evaluate a project, execute the following command from its directory:
 ```bash
-bin/console analyze:php:code-compliance --format=yaml
+bin/console analyze:php:code-compliance
```

Comment:
Why did we remove format?

Reply:
We haven't this option in current console and always use `yaml` instead